cluster/ci/config/roles.yaml

```diff
@@ -145,3 +160,16 @@ items:
     kind: Role
     apiGroup: rbac.authorization.k8s.io
     name: admin
+- kind: RoleBinding
+  apiVersion: rbac.authorization.k8s.io/v1beta1
+  metadata:
+    name: admin
+    namespace: image-registry-publishing-bot
+  subjects:
+  - kind: Group
+    apiGroup: rbac.authorization.k8s.io
+    name: image-registry-dev
+  roleRef:
+    kind: Role
```

Comment:
/hold 

I think this needs to be ClusterRole

Reply:
You are right, fixed.